veggieseasons/lib/widgets/veggie_card.dart

```diff
@@ -119,10 +123,23 @@ class VeggieCard extends StatelessWidget {
       },
       child: Stack(
         children: [
-          Image.asset(
-            veggie.imageAssetPath,
-            fit: BoxFit.cover,
-            semanticLabel: 'A card background featuring ${veggie.name}',
+          Semantics(
+            label: 'A card background featuring ${veggie.name}',
+            child: Container(
+              height: isInSeason ? 350 : 150,
+              decoration: BoxDecoration(
+                image: DecorationImage(
+                  fit: BoxFit.cover,
+                  colorFilter: isInSeason
+                      ? null
+                      // 222222 is a random color that has low color saturation.
+                      : ColorFilter.mode(Color(0xFF222222), BlendMode.saturation),
```

Comment:
I've been sticking random colors like this in `styles.dart`. Actually, ColorFilter has a const constructor, so you can probably just put the whole filter in there.

Reply:
ah sure. Done